rust_src/src/indent.rs

```diff
@@ -0,0 +1,25 @@
+use remacs_macros::lisp_fn;
+use lisp::LispObject;
+use remacs_sys;
+use remacs_sys::MOST_POSITIVE_FIXNUM;
+use remacs_sys::EmacsInt;
+
```

Comment:
It would be better if it would look like this `use remacs_sys::{EmacsInt, current_column};`